test/perl/api-gateway/cache/cache.t

```diff
@@ -241,16 +241,19 @@ X-Test: test
             -- after 3s key1 should have expired form both cache stores: local and redis
             assert(cache:get("key1") == nil, "Value in cache for key1 should have been nil but was: [" .. tostring(cache:get("key1")) .. "]")
 
+            local cjson = require "cjson"
             --4. test that the key2 still exists but key1 does not exist anymore
-            ngx.say("key1=" .. tostring(cache:get("key1")) .. ",key2=" .. tostring(cache:get("key2")))
+            local key2 = cjson.decode(tostring(cache:get("key2")))
+            ngx.say("key1=" .. tostring(cache:get("key1")) .. ",key2={name=", key2.name, ",ttl_redis=", key2.ttl_redis,
+                ",ttl_local=", key2.ttl_local,"}")
         ';
     }
 
 --- timeout: 5s
 --- request
 GET /t
 --- response_body_like eval
-['key1=nil,key2={"ttl_redis":"13","name":"Mark","ttl_local":"12"}']
+['key1=nil,key2={name=Mark,ttl_redis=13,ttl_local=12}']
```

Comment:
What Gateway/Openresty version returns this format ? Is it`1.12.x` ?

Reply:
@ddragosd it's not the format - it's that test fail when you try to compare json with json (the key  order is not guaranted) => it's not consistent so I resorted to extracting the info into dedicated fields that's all - the format is still json